src/libs/actions/IOU.js

```diff
@@ -111,8 +111,32 @@ function createIOUSplit(params) {
         });
 }
 
+/**
+ * Settles an IOU Report. As we can guarantee that the settled report was previously active,
+ * we should update the chatReport data too when calling `fetchIOUReportByID`.
+ */
+function settleIOUReport({
+    chatReportID, reportID, paymentMethodType,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, error: ''});
+    API.PayIOU({
+        reportID,
+        paymentMethodType,
+    })
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                throw new Error(`Error while settling IOU: ${response.message}`);
+            }
+        })
+        .then(fetchChatReportsByIDs([chatReportID]))
+        .then(fetchIOUReportByIDAndUpdateChatReport(reportID, chatReportID))
```

Comment:
I guess my expectation here is that `.then()` should take a function (and not pass the return value of one) so these would look like 
```suggestion
        .then(() => fetchIOUReportByIDAndUpdateChatReport(reportID, chatReportID))
```

Reply:
Ah I see. No problem 👍